examples/visualization/brain.py

```diff
@@ -0,0 +1,65 @@
+"""
+.. _ex-brain:
+
+===============================
+Plotting with ``mne.viz.Brain``
+===============================
+
+In this example, we'll show how to use :class:`mne.viz.Brain`.
+"""
+# Author: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD-3-Clause
+
+# %%
+
+import os.path as op
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import sample
+
+print(__doc__)
+
+data_path = sample.data_path()
+sample_dir = op.join(data_path, 'MEG', 'sample')
+subjects_dir = op.join(data_path, 'subjects')
+evoked = mne.read_evokeds(op.join(sample_dir, 'sample_audvis-ave.fif'))[0]
+trans = mne.read_trans(op.join(sample_dir, 'sample_audvis_raw-trans.fif'))
+stc = mne.read_source_estimate(op.join(sample_dir, 'sample_audvis-meg'))
+
+brain = mne.viz.Brain('sample', subjects_dir=subjects_dir,
+                      alpha=0.5, cortex='low_contrast')
+
+# add sensors
+# brain.add_sensors(
+#     epochs.info, trans, colors=dict(eeg='green', grad='blue', mag='purple'),
+#     ch_types=dict(meg=True, eeg=True))
```

Comment:
```suggestion
# %%
# View sensors and brain
# ----------------------
#
# To view your sensors on the brain you can use the ``add_sensors`` method.
# This can be useful to validate your data has been
# stored and retrieved correctly.

# add sensors
# brain.add_sensors(
#     epochs.info, trans, colors=dict(eeg='green', grad='blue', mag='purple'),
#     ch_types=dict(meg=True, eeg=True))
```